.github/workflows/build.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         extension: ${{ fromJSON(needs.set_matrix.outputs.matrix) }}
         include:
           - arch: amd64
-            runs-on: buildjet-4vcpu-ubuntu-2204
+            runs-on: ubuntu-latest
```

Comment:
This further brings down cost of BuildJet that for amd64 builds, we are using the free GitHub Actions runner for public rep :)